### PR TITLE
chore: Fill in since annotation for past releases

### PR DESCRIPTION
**Pull request by NotMyFault**
```
 ➜  jenkins git:(chore/master/fill-in-since) ./update-since-todo.py 
Analyzing core/src/main/java/hudson/model/Descriptor.java:602
        first sha: 8f6e04bcb51081627c11df2a105bcb049a1c8339
        first tag was jenkins-2.342
        Updating file in place
Analyzing core/src/main/java/hudson/model/UpdateSite.java:533
        first sha: b5023f6bc7b3be31fdff34b539b51434778fc6a6
        first tag was jenkins-2.343
        Updating file in place

List of commits introducing new API and the first release they went in:
* https://github.com/jenkinsci/jenkins/releases/tag/jenkins-2.342
  - https://github.com/jenkinsci/jenkins/commit/8f6e04bcb51081627c11df2a105bcb049a1c8339
* https://github.com/jenkinsci/jenkins/releases/tag/jenkins-2.343
  - https://github.com/jenkinsci/jenkins/commit/b5023f6bc7b3be31fdff34b539b51434778fc6a6
```

### Proposed changelog entries

* N/A, skip changelog

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [x] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/6682"></a>

### CODE REVIEW

**Review by MarkEWaite — APPROVED** [review]

**Review by basil — APPROVED** [review]
This PR is now ready for merge. We will merge it after approximately 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process. Thanks!

**Comment by NotMyFault** [comment]
Would someone mind if I force-push correct the commit message to get a rid of the user mention there? lol

**Pull request merged**
